File: week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8); //ensure a valid input

    for (int i = 1; i < h + 1; i++)
    {
        for (int j = 1; j < h - i + 1; j++)
        {
            printf(" ");
        }

        for (int k = 1; k < i + 1; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int k = 1; k < i + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: week6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    n = get_int('Height: ')
    if 0 < n < 9:
        break
    else:
        print('Please type a int between 1 and 8')

for row in range(n):
    print(' '*(n - row - 1), end='')
    print('#'*(row + 1), end='')
    print('  ', end='')
    print('#'*(row+1))